README.md

```diff
@@ -54,3 +79,8 @@ Now in your browser open files index.html and index2.html
 	Made with :heart: by DSC VIT
 </p>
 
+
+[![saythanks](https://img.shields.io/badge/say-thanks-ff69b4.svg)](https://github.com/D-E-F-E-A-T) 
```

Comment:
None of these badges are required. Just one for documentation and other for the link